client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import ClipboardButtonInput from 'components/clipboard-button-input';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import { getPurchase, isDataLoading } from '../../utils';
+import { isJetpackPlan, isFreeJetpackPlan } from 'lib/products-values';
+import { getPluginsForSite } from 'state/plugins/premium/selectors';
+
+class PurchasePlanDetails extends Component {
+	renderPluginLabel( slug ) {
+		switch ( slug ) {
+			case 'vaultpress':
+				return this.props.translate( 'Backups and security scanning API key' );
+			case 'akismet':
+				return this.props.translate( 'Anti-spam API key' );
+		}
+	}
+
+	render() {
+		if ( isDataLoading( this.props ) || ! this.props.selectedSite ) {
+			return null;
+		}
+
+		const { selectedSite, pluginList } = this.props;
+		const purchase = getPurchase( this.props );
+		if ( ! isJetpackPlan( purchase ) || isFreeJetpackPlan( purchase ) ) {
+			return null;
+		}
+
+		return (
+			<Card>
+				<QueryPluginKeys siteId={ selectedSite.ID } />
+				{ pluginList.map( ( plugin, i ) => {
+					return (
+						<FormFieldset key={ i }>
```

Comment:
A design item really, but there's some extra space below the last `FormFieldset`:

![](https://cldup.com/TBdDcsY3rQ.png)

Can we avoid that by setting a `margin-bottom: 0` of the `.form-fieldset:last-child`?